lib/modules/ai_bot/commands/command.rb

```diff
@@ -36,11 +39,25 @@ def parameters
           end
         end
 
-        attr_reader :bot_user, :args
+        attr_reader :bot_user
 
-        def initialize(bot_user, args)
+        def initialize(bot_user:, args:, post: nil, parent_post: nil)
           @bot_user = bot_user
           @args = args
+          @post = post
+          @parent_post = parent_post
+
+          @placeholder = +(<<~HTML).strip
+            <details>
+              <summary>#{I18n.t("discourse_ai.ai_bot.command_summary.#{self.class.name}")}</summary>
+              <p>
+                #{CARET}
+              </p>
+            </details>
+            #{CARET2}
```

Comment:
Maybe we just have some text before the dots like `Generating image, please wait...` then just keep adding dots to the end.

Reply:
cool ... will rename a bit ... agree caret 2 is somewhat confusing

Reply:
generating image ... is a bit of a problem cause one big point of this is so you see the prompt early. 

"Generating: "Amazing robots flying" ... " could work though ... will give it a shot ...